sarahbenmessaoud.svg

```diff
@@ -0,0 +1,106 @@
+<svg viewBox="0 -25 400 200" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you remove this file? You can do that by going to your fork, to this file, and then clicking the trash icon in the top-right corner.